healthcare.py

```diff
@@ -0,0 +1,1949 @@
+# print ("sample-weighted Beta score With a F(2), emphasis on Recall is ",fbeta_score(y_test_validation, y_predicted_ml_RandomForestClassifier, average='weighted', beta=2.0))
+# print ("sample-weighted Beta score With a F(0.5), emphasis on Precision is ",fbeta_score(y_test_validation, y_predicted_ml_RandomForestClassifier, average='weighted', beta=0.5))
+
+# ###Conclusion 
+#     ##The model predicted 63  people Not to have a stroke & they had a stroke. 
+#     ##The model predicted 1140  people Not to have a stroke & they did not have a stroke. 
+#     ##The model predicted 4 people to have a stroke & they had a stroke. 
+#     ##The model predicted 10 to have a stroke and they did not have a stroke. 
+#     ##For "0" Not having a stroke ---> precision =95%, recall =  99%, f1-score = 97% 
+#     ##For "1"  having a stroke ---> precision =29%, recall =  6%, f1-score = 10.00% 
+#     ##sample-weighted F1 score is  0.9210677287568355
+#     ##sample-weighted Beta score With a F(2), emphasis on Recall is    0.9320788143903541
+#     ##sample-weighted Beta score With a F(0.5), emphasis on Precision is   0.9123727003311718
+
+'''
+Section 6.6.2.3 
+-----------
+Based on the Random Forest Classification Model Evaluate it For Accuracy
+'''
+# from sklearn.ensemble import RandomForestClassifier
+
+
+# ml_RandomForestClassifier = RandomForestClassifier(bootstrap=True, max_depth=None, max_features='sqrt', n_estimators=400, n_jobs=-1)
+# ml_RandomForestClassifier.fit(X_train_sm_balanc1,y_train_sm_balanc1 ) #fitting the model to the X and y trained data
+
+# y_predicted_ml_RandomForestClassifier= ml_RandomForestClassifier.predict(X_test_validation) #view the predicted results from the fitted model by predicting the X Test (X_validation)
+
+# from sklearn.metrics import ConfusionMatrixDisplay
+# from sklearn.metrics import confusion_matrix,classification_report
+# cm = confusion_matrix(y_test_validation, y_predicted_ml_RandomForestClassifier, labels=ml_RandomForestClassifier.classes_)
+# disp = ConfusionMatrixDisplay(confusion_matrix=cm, display_labels=ml_RandomForestClassifier.classes_)
+# disp.plot()
+# plt.show() 
+
+# ##View the classification report 
+# ML_logistic_model_classification_report  = classification_report(y_test_validation,y_predicted_ml_RandomForestClassifier)
+# print(ML_logistic_model_classification_report)
+# from sklearn.metrics import f1_score, fbeta_score
+# print("sample-weighted F1 score is ", f1_score(y_test_validation, y_predicted_ml_RandomForestClassifier, average='weighted'))
+# print ("sample-weighted Beta score With a F(2), emphasis on Recall is ",fbeta_score(y_test_validation, y_predicted_ml_RandomForestClassifier, average='weighted', beta=2.0))
+# print ("sample-weighted Beta score With a F(0.5), emphasis on Precision is ",fbeta_score(y_test_validation, y_predicted_ml_RandomForestClassifier, average='weighted', beta=0.5))
+
+# ###Conclusion 

```

Comment:
Should include the confusion matrix, it is faster to read than this text.

Reply:
I have a confusion matrix for the TP, TN, FP, and FN. However, i will consider some sort of a matrix for finding the fbeta